ProcessSlave.php

```diff
@@ -384,18 +391,20 @@ protected function handleRequest(\React\Http\Request $request, HttpResponse $res
             );
             $this->shutdown();
         }
+        return $response;
     }
 
-    protected function prepareEnvironment(\React\Http\Request $request)
+    protected function prepareEnvironment(RequestInterface $request)
     {
         $_SERVER = $this->baseServer;
```

Comment:
The same thing i was thinking about today, but decided to check it later ;)

IMO this is bad when we enable concurrent requests, requests contexts can be mixed up. Didn't check this yet, but will add this to tasks.